From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Wed, 5 Apr 2023 11:33:40 -0600
Subject: [PATCH] kernelbase: HACK: Add --use-gl=swiftshader for Planetside
 2/LaunchPad.exe.

CW-Bug-Id: #22083
Valve-commit: d814ac3cface480acbd6d72b6af8f8b08d20fafa
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index fdfc07a..d05e491 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -516,6 +516,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
         {L"OlympiaRising.exe", L" --use-gl=swiftshader"},
         {L"nw.exe.exe", L" --use-angle=d3d9"},
         {L"DC Universe Online\\LaunchPad.exe", L" --use-gl=swiftshader"},
+        {L"PlanetSide 2\\LaunchPad.exe", L" --use-gl=swiftshader"},
     };
     unsigned int i;
 
